spec/index.bs

```diff
@@ -1894,6 +1900,111 @@ dictionary GPUVertexAttributeDescriptor {
 };
 </script>
 
+## {{GPUPipelineBase/getBindGroupLayout()|getBindGroupLayout}} ## {#get-bind-group-layout}
+
+<dl dfn-type="method" dfn-for="GPUPipelineBase">
+    : <dfn>getBindGroupLayout(unsigned long index)</dfn>
+    ::
+        <div algorithm="GPUProgrammablePipeline.getBindGroupLayout(unsigned long index)">
+            1. If |index| is greater or equal to {{GPULimits/maxBindGroups}} :
+
+                1. Throw a {{RangeError}}.
+
+            1. If |this|.getBindGroupLayout has already been called with |index|:
```

Comment:
what about the case where the pipeline was created with an explicit layout? where are we handling this?

Reply:
Whoops I completely forgot that ^^